src/backend/distributed/sql/citus--9.0-2--9.1-1.sql

```diff
@@ -8,3 +8,7 @@ COMMENT ON COLUMN pg_catalog.pg_dist_node.shouldhaveshards IS
 -- we don't maintain replication factor of reference tables anymore and just
 -- use -1 instead.
 UPDATE pg_dist_colocation SET replicationfactor = -1 WHERE distributioncolumntype = 0;
+
+-- drop function which was used for upgrading from 6.0
+-- creation was removed from citus--7.0-1.sql
+DROP FUNCTION IF EXISTS master_initialize_node_metadata;
```

Comment:
Should to be careful about search_path here. We've made mistakes where because we use "IF EXISTS", we did not notice that we were not dropping the function. That caused pg_upgrade to fail, since pg_upgrade will try to re-create the function the new database, but that will fail if the C implementation is missing.

Reply:
Updated to qualified, `DROP FUNCTION IF EXISTS pg_catalog.master_initialize_node_metadata;`